src/contract.rs

```diff
@@ -6,14 +6,16 @@ mod deploy;
 mod method;
 
 use crate::errors::{DeployError, LinkError};
-use crate::truffle::abi::Result as AbiResult;
-use crate::truffle::{Abi, Artifact, Bytecode};
+use ethcontract_common::abi::Result as AbiResult;
+use ethcontract_common::truffle::Network;
```

Comment:
Is this still truffle specific or should this also be lifted into common?

Reply:
It is truffle specific.